src/json/UnitPrettyValueFormatContext.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Unit Pretty Value Format Context",
+  "description": "The context for rendering custom unit prettyValue",
+  "type": "object",
+  "properties": {
```

Comment:
@00salmon and @jorgecgll FYI these are the variables you can use for creating custom `prettyValue` formats

Examples:
 - `{value} Points`
 - `{value} {unit}`
 - `{currencyLocalizedSymbol}{valueInCurrencyBaseUnit}`